packages/fiori/test/pages/ViewSettingsDialog.html

```diff
@@ -88,11 +88,21 @@ <h3> ViewSettingsDialog</h3>
 		</ui5-filter-item>
 	</ui5-view-settings-dialog>
 
+	<br>
+	<br>
+	<ui5-input id="sortByItem" style="display: none"></ui5-input>
+	<ui5-input id="sortOrder" style="display: none"></ui5-input>
+
 	<script>
 		btnOpenDialog.addEventListener("click", function () {
 			vsd.show();
 		});
-		vsd.addEventListener("confirm", function(evt) {
+		vsd.addEventListener("ui5-confirm", function(evt) {
+			const sortByItemTagName = evt.detail.sortByItem && evt.detail.sortByItem.tagName;
+			const isSortDescending = evt.detail.sortDescending;
+			sortByItemTagName && document.getElementById("sortByItem").setAttribute("value", sortByItemTagName);
+			document.getElementById("sortOrder").setAttribute("value", isSortDescending);
```

Comment:
You can just write
```js
sortByItem.value = sortByItemTagName
```
instead of
```js
document.getElementById("sortByItem").setAttribute("value", sortByItemTagName);
```

and
```js
sortOrder.value = isSortDescending
```
instead of
```js
document.getElementById("sortOrder").setAttribute("value", isSortDescending);
```

Reply:
done